src/tests/test_main.py

```diff
@@ -187,6 +189,18 @@ def test_expand_uris(self):
         ]
         self.assertEqual(main.expand_uris(msid, given), expected)
 
+    def test_expand_inline_graphics(self):
```

Comment:
I'm not sure the inline graphics should go through IIIF rather than just the CDN. The <img> tag doesn't look like it can support a full-fledged triple IIIF viewer? The URL in `src` would redirect to the `info.json` image descriptor rather than returning the image content.